swap/src/cli.rs

```diff
@@ -11,4 +11,9 @@ pub struct Options {
     /// Run the swap as Bob and try to swap this many BTC (in satoshi).
     #[structopt(long = "sats")]
     pub satoshis: Option<u64>,
+
+    /// Alice's onion multitaddr (only required for Bob, Alice will autogenerate
+    /// one)
```

Comment:
This comment is not clear to me. If Alice autogenerates an address how is Bob supposed to know it?

Reply:
Bob needs to know the address of "an Alice" to trade with her. So yes, Alice needs to communicate that with her.